routes/admin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+use Illuminate\Support\Facades\Route;
+
+use App\Http\Controllers\ClassesController;
+
+
+Route::group([
+    'controller'=>ClassesController::class,
+    'prefix'=>'classes',
+    'as'=>'classes.',
+    //apply custom middleware isAdmin + custom routes 
+    'middleware'=>['web','verified','isAdmin','auth'],
```

Comment:
yeah that it, good job ya @esraa33-95, one thing to note
`auth` is used to check if the user is logged in 
`verified` check user login and  verified

don't you think that `auth` middleware is redundant?